app/controllers/admin/categories_controller.rb

```diff
@@ -14,16 +15,21 @@ def create
     end
   end
 
+  def edit
+    @category = Category.find(params[:id])
+    authorize! :edit, @category
+  end
+
   def update
     @category = Category.find(params[:id])
+    authorize! :update, @category
     @item = Item.find(params[:item_id])
     @category.items.delete(@item)
     flash[:notice] = "Successfully Removed Item from #{@category.name}"
     redirect_to categories_path
   end
 
-  private
-
+private
```

Comment:
Indent access modifiers like `private`.<br>Keep a blank line before and after `private`.